src/sagas/pushNotification.js

```diff
@@ -0,0 +1,432 @@
+/* eslint-disable max-len */
+import {
+  HathorWalletServiceWallet,
+  PushNotification as pushLib,
+  wallet as walletUtil,
+  Network,
+  config,
+} from '@hathor/wallet-lib';
+import moment from 'moment';
+import {
+  put,
+  takeEvery,
+  all,
+  call,
+  select,
+  race,
+  take,
+  fork,
+} from 'redux-saga/effects';
+import messaging from '@react-native-firebase/messaging';
+import notifee from '@notifee/react-native';
+import { t } from 'ttag';
+import {
+  types,
+  pushRegisterSuccess,
+  pushRegisterFailed,
+  pushUpdateSuccess,
+  pushUpdateFailed,
+  pushUpdateDeviceId,
+  pushRegistrationRequested,
+  setIsShowingPinScreen,
+  pushLoadWalletRequested,
+  pushLoadWalletSuccess,
+  pushLoadWalletFailed,
+  pushInit,
+} from '../actions';
+import {
+  pushNotificationKey,
+  STORE,
+  WALLET_SERVICE_MAINNET_BASE_WS_URL,
+  WALLET_SERVICE_MAINNET_BASE_URL,
+} from '../constants';
+import NavigationService from '../NavigationService';
+import { getPushNotificationSettings } from '../utils';
+
+export const PUSH_API_STATUS = {
+  READY: 'ready',
+  FAILED: 'failed',
+  LOADING: 'loading',
+};
+
+/**
+ * this is the network name that will be used to load the wallet on the wallet-service,
+ * it is first hardcoded in the `startWallet` saga function, @see src\sagas\wallet.js.
+ */
+const NETWORK = 'mainnet';
```

Comment:
Maybe we could have this in the constants file and import from there. What do you think? So when we need to change to testnet, then we change only in one place.

Reply:
Good!